docker-compose.yml

```diff
@@ -111,10 +111,12 @@ services:
       # settings reference: https://neo4j.com/docs/operations-manual/4.4/docker/ref-settings/
       # TODO: This sounds scary for a production environment
       - NEO4J_AUTH=none
+      # - NEO4J_dbms_default__database=graph.db
```

Comment:
this works?

Reply:
This works like we need it.
I went with greate energy to find out how we can achive the solution you sugested in an easy way.

If you set this name the dump command uses this database as default if you have no name given in the command.

If this database doesn't exist on starting Neo4j a database with this name gets created and is empty.
The other databases are not effected and you can use them again by stopping Neo4j and restart it with the database name you like to use.

I tested it locally very careful, because we don't want to make any harm.

If you like @ulfgebhardt lets merge this?